testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -4667,9 +4695,12 @@ protected TestTable createTableWithOneIntegerColumn(String namePrefix)
     public void testUpdateWithPredicates()
     {
         if (!hasBehavior(SUPPORTS_UPDATE)) {
-            // Note this change is a no-op, if actually run
-            assertQueryFails("UPDATE nation SET nationkey = nationkey + regionkey WHERE regionkey < 1", MODIFYING_ROWS_MESSAGE);
-            return;
+            if (!hasBehavior(SUPPORTS_ROW_LEVEL_UPDATE)) {
+                // Note this change is a no-op, if actually run
+                assertQueryFails("UPDATE nation SET nationkey = 100 WHERE regionkey = -1", MODIFYING_ROWS_MESSAGE);
+                return;
+            }
+            throw new SkipException("Row level update supported");
```

Comment:
use `skipTestUnless(!hasBehavior(SUPPORTS_ROW_LEVEL_UPDATE))`. Update this also in other place.

Reply:
refactored